RELEASE_NOTES.txt

```diff
@@ -2,6 +2,238 @@ Apache Ignite Release Notes
+* Fixed speed base throttling unexpectedly degraded to zero
+* Fixed spring data integration which doesn't introspect the fields of the key object
+* Fixed starvation in mgmt pool caused by  MetadataTask execution
+* Fixed synchronization issues when different classloaders are used for deployment of same class
+* Fixed transaction commit completes successfully if partition is lost on commit phase
+* Fixed validate index does not stop after process of control.sh was interrupted
+* Fixed validateKeyAndValue which attempts to deserialize key and value when QueryEntity.fields is not set
+* Fixed waiting for completion of operations on indexes before cache stop
+* Moved ClusterMetrics to the new metrics framework
+* Moved TcpDiscoveryStatistics to new metrics framework
+* Moved spring-data modules to ignite-extensions
+* Removed 'default' cache usage from REST API
+* Removed WALPointer and use FileWALPointer
+* Removed ability to delete segments from the middle of WAL archive
+* Removed unnecessary dependency to curator-client from ZookeeperDiscoverySpi
+* Removed unneeded ZkPinger from ZookeeperDiscovery
+
+SQL:
+* Added lazy flag to synthetic SELECT used by first step of DML query
+* Added tracing of SQL queries
+* Added unification of fallback properties for SSL settings as per Java standard
+* Fixed REPLICATED cache cannot be left-joined to PARTITIONED
+* Fixed decimal columns in SQL result set have invalid precision and scale
+* Fixed exception which occurs on SQL caches when client reconnect
+* Fixed tracing of SELECT queries causes incorrect span inheritance
+* Fixed transformation of sub-queries to JOINs when possible
+
+JDBC/ODBC:
+* Added ODBC SQLNumResultCols for prepared statement
+* Added ODBC driver assigns SQL_BINARY type to DATE fields
+* Added ODBC driver shows real nullability info
+* Added ODBC implementation of SQLRowCount for select queries
+* Added ODBC support of SQL_ATTR_ROW_ARRAY_SIZE with value more than one
+* Fixed thin driver reports incorrect property names
+
+Java thin-client:
+* Added non-blocking socket IO usage
+* Added support for IPv6 addresses
+* Added thin Client Continuous Query
+* Added thin client Java API - async API
+* Added thin client Kubernetes discovery
+* Fixed CHM get/remove on each request to server
+* Fixed connect/disconnect during topology update may lead to partition divergence in ignite-sys-cache
+* Fixed exception type is lost on rethrow
+* Fixed message not fully read by client after SECURITY_VIOLATION error
+* Fixed thin client silently closes channel after inactivity
+* Fixed thin clients cannot access the Ignite Service deployed through UriDeploymentSpi
+* Fixed unsupported protocol version exception when getting cache configuration from Java thin client
+
+Thin-client:
+* Moved Node.js thin client to a separate git repo
+* Moved PHP thin client to a separate git repo
+* Moved Python thin client to a separate git repo
+
+Ignite .NET:
```

Comment:
My point below is to highlight that the issue is resolved (i.e., it no longer exists). So I suggest sticking to the past tense.

Reply:
Agree, I'll recheck.